stack-package/.registry/app.yaml

```diff
@@ -32,6 +32,8 @@ keywords:
 - "examples"
 - "tutorials"
 
+permissionScope: "Namespaced"
```

Comment:
i think we'll need to make the controller manager only watch in a single namespace, similar to what we did for wordpress: https://github.com/crossplaneio/sample-stack-wordpress/pull/12/files

Reply:
@displague do we need to get these metadata changes in now for any reason, and deal with the runtime changes to watch resources in a single namespace in a follow-up PR? or do you want to do it all in this PR?